analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Noncompliant.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+
+public class Nullable
+{
+
+}
+
+public class BlazorQueryParameterRoutableComponent_Noncompliant : ComponentBase
+{
+    [Parameter]
+    [SupplyParameterFromQuery]
+    public int Number { get; set; } // Noncompliant {{Component parameters can only receive query parameter values in routable components.}}
+    //         ^^^^^^
+
+    [Parameter]
+    [SupplyParameterFromQuery]
+    public string MyString { get; set; } // Noncompliant    [Parameter]
```

Comment:
Not sure what's this
```suggestion
    public string MyString { get; set; } // Noncompliant
```

Reply:
Probably bad merging